concrete/src/Html/Image.php

```diff
@@ -28,8 +29,10 @@ class Image
     /**
      * @param \Concrete\Core\Entity\File $f
      * @param bool|null $usePictureTag
+     * @param bool|null $lazyLoadNative
+     * @param bool|null $lazyLoadJavaScript
      */
-    public function __construct(File $f = null, $usePictureTag = null)
+    public function __construct(File $f = null, $usePictureTag = null, $lazyLoadNative = null, $lazyLoadJavaScript = null)
```

Comment:
Instead of a long parameter tail, what about using an array to hold the options?

To make it backward compatible, we can use an approach like this:

```php
/**
 * @param \Concrete\Core\Entity\File\File $f
 * @param array|bool|null Use NULL to use the default options, a boolean (deprecated) to use a `picture` tag (TRUE) or an `img` tag (FALSE), or an array with these values:
 * <ul>
 *     <li>bool|null `usePictureTag`: TRUE (use a `picture` tag), FALSE (use an `img` tag), NULL or not specified to use the settings from the current page theme</li>
 *     <li>bool|null `lazyLoadNative`: ...</li>
 *     <li>bool|null `lazyLoadJavaScript`: ...</li>
 * </ul>
 */
public function __construct(File $f = null, $options = null)
{
    if ($options === null) {
        $options = [];
    } elseif (!is_array($options)) {
        $options = [
            'usePictureTag' => $options,
        ];
    }
    $options += [
        `usePictureTag` => null,
        `lazyLoadNative` => null,
        `lazyLoadJavaScript` => null,
    ];

    if ($options['usePictureTag'] !== null) {
        $this->usePictureTag = $options['usePictureTag'];
    } else {
        $this->loadPictureSettingsFromTheme();
    }
    // ...
}
```

Reply:
I like @mlocati's  array options suggestion. Keeps a cleaner interface and avoids a growing tail should further options be added in the future.
Also makes it cleaner to override in other uses.

Reply:
@mlocati and @JohntheFish

Thank you very much for the suggestion, this is definitely an improvement.